app-tests/next-button/src/pages/_app.js

```diff
@@ -1,7 +1,15 @@
+import React from 'react';
+import pt from 'prop-types';
+
 import '@espressive/legacy-css';
 
-const MyApp = ({ Component, pageProps }) => {
-  return <Component {...pageProps} />;
+const propTypes = {
+  Component: pt.element,
+  pageProps: pt.shape({}),
```

Comment:
Same with this. Let's actually use `object` and comment why when we disable the rule